demo.yml

```diff
@@ -50,36 +42,46 @@ x-services:
       - internal
     env_file:
       - ./demo-default.env
-      - ./demo-local.env
+      - ./demo-${COMPOSE_PROJECT_NAME}.env
+      - ./demo-${COMPOSE_PROJECT_NAME}-local.env
 
   base_api: &base_api_service
     <<: *base_service
     networks:
       - apis_external
       - internal
 
+
 services:
-  # service providers - minio, elasticMQ, postgres
+  ##
+  ## service providers - minio, elasticMQ, postgres
+  ##
+
+  # needed only for easy channels development - because on Linux you can't
+  # access the host container in any other reliable way
+  # And shared channels are usually started on the host for the local/demo
+  docker-host:
+    image: qoomon/docker-host
+    cap_add: [ 'NET_ADMIN', 'NET_RAW' ]
+    restart: on-failure
+    networks:
+      - internal
+
   postgresql:
     image: postgres:10.5
     environment:
       - POSTGRES_USER=intergovuser
       - POSTGRES_PASSWORD=intergovpassword
-    # we don't have to expose it directly to the host machine for our demo,
-    # but it's useful for starting service on the host
-    # ports: ['5432:5432']
     networks:
       - internal
+    volumes:
+      - postgresql_data:/var/lib/postgresql/data
     restart: on-failure
 
   elasticmq:
     image: softwaremill/elasticmq
-    # we don't have to expose it directly to the host machine for our demo,
-    # but it's useful for starting service on the host
-    ports: ['9324:9324']
     volumes: ['./tests/local_elasticmq.conf:/opt/elasticmq.conf']
     logging:
-      # because it's noisy
```

Comment:
Was this the reason for logging being disabled for Postgres? It made it painful to figure out why postgres wouldn't start for me. Changing the line and restarting the postgres service didn't work - I had to down and then up the whole stack for it to re-read the change :-(

Reply:
:-(
I hope you won't need these logs when it's working well :-)